### ISSUE TITLE
"Sequence contains no matching element" using GenerateEntities on Oracle DB

### ISSUE THREAD

**Issue by rucquoy** [Apps72/Dev.Data#32]
Using the following command 

```
DbCmd GenerateEntities -cs="MY_CORRECTLY_FORMATTED_CONNECTION_STRING" --provider=Oracle --Namespace="MySolution.MyProject.Model" --NullableRefTypes --SortProperties
```
Yields an error "Sequence contains no matching element". 

Similar issues have been reported using EntityFramework (see https://stackoverflow.com/questions/22982749/sequence-contains-no-matching-element-entityframework). It could be a name issue for column types. 

Could you please check this ?

**Comment by dvoituron** [comment]
Thanks for your feeback. I'll check that with a sample Oracle database

**Comment by dvoituron** [comment]
This bug was fixed in the version 4.1.2 of this tools (see https://www.nuget.org/packages/Apps72.Dev.Data.Generator.Tools).
But the problem may be occurred when the list of tables is to large.
So, you need to add the flag `--OnlySchema` to retrieve only some tables.
